### PR TITLE
[10.0] add new module account_financial_report_date_range

### PR DESCRIPTION
**Pull request by luc-demeyer**
This module adds the 'Date Range' field to the Odoo CE standard addons financial reports wizard.

### CODE REVIEW

**Comment by ovnicraft** [comment]
Good for core reports.
LGTM.
Anyone else review this and merge ?

File: account_financial_report_date_range/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+{
+    'name': 'Date Range Year on Financial Reports',
+    'summary': """
+        Add Date Range field to the Odoo OE standard addons
+        financial reports wizard.
+    """,
+    'version': '10.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'website': 'https://github.com/OCA/account-financial-tools',
```
Comment (alexis-via) [inline account_financial_report_date_range/__manifest__.py]: Wrong projet name

**Review by alexis-via — APPROVED** [review]
Code review, no test

File: account_financial_report_date_range/wizards/accounting_report.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+  <record id="accounting_report_view" model="ir.ui.view">
+    <field name="name">accounting.report.form.date_range</field>
+    <field name="model">accounting.report</field>
+    <field name="inherit_id" ref="account.accounting_report_view"/>
+    <field name="arch" type="xml">
+      <field name="date_from" position="before">
+        <field name="date_range_id" options="{'no_create_edit': True}"/>
```
Comment (ovnicraft) [inline account_financial_report_date_range/wizards/accounting_report.xml]: as plus include `no_create: True`

**Comment by ovnicraft** [comment]
Solving comments LGTM, code reviewed

**Comment by ovnicraft** [comment]
💯

**Pull request merged**